DependencyInjection/Compiler/AspectCollectorPass.php

```diff
@@ -19,16 +19,15 @@
  */
 class AspectCollectorPass implements CompilerPassInterface
 {
-
     /**
-     * You can modify the container here before it is dumped to PHP code.
-     *
-     * @param ContainerBuilder $container
-     *
-     * @api
+     * {@inheritdoc}
      */
     public function process(ContainerBuilder $container)
     {
+        if (!$container->hasDefinition('goaop.aspect.container')) {
```

Comment:
Could you please describe how it's possible not to have this definition? If we loaded our bundle and services from the XML, then current compiler pass will always have `'goaop.aspect.container'` definition.